### PR TITLE
[WIP] Removed windows dep and unify shippable.sh

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Rebased version of #35.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

File: tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 - name: postgresql_ext - install postgis on Linux
   package: name=postgis state=present
   when: ansible_os_family != "Windows"
-- name: postgresql_ext - install postgis on Windows
```
Comment (gundalow) [inline tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml]: @Andersson007 community.general isn't tested against Windows CI targets, so we are removing this. Is that OK with you?

Comment (Andersson007) [reply tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml]: @gundalow , yep, sure

Comment (Andersson007) [reply tests/integration/targets/postgresql_ext/tasks/postgresql_ext_initial.yml]: (it is probably something obsolete)

**Comment by felixfontein** [comment]
The errors already look much better. (Also, there are random network problems when installing the dependent collections. Something new :) )

File: tests/utils/shippable/shippable.sh [hunk 2]
```diff
@@ -29,8 +29,36 @@ python -V
...
+ansible-galaxy collection install community.kubernetes
+ansible-galaxy collection install netbox.netbox
+ansible-galaxy collection install netapp.ontap
+ansible-galaxy collection install cisco.meraki
+ansible-galaxy collection install fortinet.fortios
+ansible-galaxy collection install junipernetworks.junos
+ansible-galaxy collection install cisco.aci
+ansible-galaxy collection install google.cloud
+ansible-galaxy collection install community.kubernetes
+ansible-galaxy collection install f5networks.f5_modules
```
Comment (gundalow) [inline tests/utils/shippable/shippable.sh]:
```suggestion
ansible-galaxy -vvv collection install ansible.posix
ansible-galaxy -vvv collection install community.crypto
ansible-galaxy -vvv collection install ansible.netcommon

# unit tests
ansible-galaxy -vvv collection install community.kubernetes
ansible-galaxy -vvv collection install netbox.netbox
ansible-galaxy -vvv collection install netapp.ontap
ansible-galaxy -vvv collection install cisco.meraki
ansible-galaxy -vvv collection install fortinet.fortios
ansible-galaxy -vvv collection install junipernetworks.junos
ansible-galaxy -vvv collection install cisco.aci
ansible-galaxy -vvv collection install google.cloud
ansible-galaxy -vvv collection install community.kubernetes
ansible-galaxy -vvv collection install f5networks.f5_modules
```

**Comment by felixfontein** [comment]
@jborean93 @gundalow thanks a lot!

**Pull request merged**